### ISSUE TITLE
Fake content treatment

### ISSUE THREAD

**Issue by bart-d** [16colo-rs/16c#47]
I'm looking for feedback on how to treat certain content on 16c. We already have the following functions available that modify how certain content is (not) shown:

1. blacklisting: the item will not be displayed nor be downloadable (including the pack), intended for balantly illegal content
2. hiding: hide clutter files (such as repetitive ads, dividers, ...) from the default view. visitor can override
3. nsfw tag: items show up as blurred, visitor has to click through to see the content

There is however content that is considered not to be genuine and there are multiple variants. The most obvious ones would be be (partially) ripped content or content that was signed with someone else's handle. There are also examples of entirely fake packs (like acdu0996), some packs are mostly fake but do contain some genuine content (reunion packs).

Some packs are to be considered parodies and modifying displaying does not apply here, despite how someone may 'dislike' their content, as long as there is no forgery or (edge) legal issues, they should be shown as they are (eg dd, mistergirls, fool).

I believe 'deleting content' is not desirable (but do tell me if you feel like that). I'm looking for feedback/ideas on how to keep the content for historical reason but to have a way of making clear what the visitor is dealing with.  That could be a warning message which can exist in various format, just static or clickthrough (like nsfw). The message could free text (leaves that to an editor) or a semi-static list of reasons which can be ticked by the editor. eg: this work is not displayed because 1. it's ripped 2. it's signature is fake 3. blah blah..

There's also a difference between an entire fake pack and a pack with some unathorized content, I wonder on which levels (pack/item) the warnings should be shown, there might also be a different way of handling things for ripped content (the primal energy rip for example is historically relevant) as opposed to balantly fake signature (reunion). I'm obviously open to any other ideas.

**Comment by andyherbert** [comment]
I think they should be displayed unless they're reasonably considered hateful towards a person or group of people, and do what YouTube does with certain channels e.g. "This broadcaster is partly funded by the Russian government", or in the case of ANSI art "This ANSI is a copy of this original work". Don't pass judgment, but provide a bit more information than can be conveyed with a tag

**Comment by christianvozar** [comment]
I agree with @andyherbert to provide more information when possible to inform the viewer of what they might be seeing. With parodies I think there is nothing wrong with adopting a "parody" tag for the editors to use. In either the parody or "ripped"/derived cases I wonder if 16c could be enhanced to input a reference (ex: for the Methane parody in Fool perhaps there is a reference to Propane's artist page and a tag of `parody` in the content. For derived works the original work could be specified so the viewer could see the original.

**Comment by hlotvonen** [comment]
I suggest having two types of additional metadata:
1. "Content / trigger warning" setting which could be turned on and if it's turned on the editor would also have to specify in more detail the reason for the warning. I think it should be a free text input for the editor to fill. It could blur the view in a similar fashion to the nsfw tag and have a button to view.
2. "Additional info" text field, which could be used for any kind of extra information that can't be conveyed by tags, like some of the things you listed in your original post. Could also be useful for any other historical info that could give more context to the files / art.
